client/src/components/form-inputs/MultiSelect.tsx

```diff
@@ -0,0 +1,100 @@
+import * as React from 'react';
+import { FormOptionValue } from '@konomi.ai/c-form';
+import {
+  Box,
+  Chip,
+  FormControl,
+  ListItemText,
+  ListItemIcon,
+} from '@mui/material';
+import CheckIcon from '@mui/icons-material/Check';
+import CloseIcon from '@mui/icons-material/Close';
+import {
+  Control,
+  Controller,
+  FieldValues,
+  UseFormSetValue,
+} from 'react-hook-form';
+import { isNonEmptyArrayOfStrings } from '../../helpers';
+import { StyledInputLabel, StyledSelect, StyledMenuItem } from './styled';
+
+export interface ISelectProps {
+  control: Control;
+  setValue: UseFormSetValue<FieldValues>;
+  name: string;
+  label: string;
+  required?: boolean;
+  options: FormOptionValue[];
+}
+
+export const MultiSelect = ({
+  control,
+  setValue,
+  name,
+  label,
+  required = false,
+  options,
+}: ISelectProps) => {
+  const handleDeleteOnChip = (
+    e: React.MouseEvent,
+    value: string,
+    selected: string[],
+  ) => {
+    e.preventDefault();
+    const filtered = selected.filter((s) => s !== value);
+    setValue(name, filtered);
+  };
+
+  return (
+    <Controller
+      name={name}
+      control={control}
+      rules={{ required }}
+      defaultValue={[]}
+      render={({ field: { onChange, value }, fieldState: { invalid } }) => (
+        <FormControl fullWidth>
+          <StyledInputLabel id={name}>{label}</StyledInputLabel>
+          <StyledSelect
+            multiple
+            label={label}
+            labelId={name}
+            value={value}
+            error={invalid}
+            onChange={onChange}
+            renderValue={(selected) => {
+              if (!isNonEmptyArrayOfStrings(selected)) return undefined;
+              return (
+                <Box sx={{ display: 'flex', flexWrap: 'wrap', gap: 0.5 }}>
+                  {selected.map((option) => (
+                    <Chip
+                      key={option}
+                      label={option}
+                      clickable
+                      deleteIcon={
+                        <CloseIcon onMouseDown={(e) => e.stopPropagation()} />
+                      }
+                      onDelete={(e) => handleDeleteOnChip(e, option, selected)}
+                    />
+                  ))}
+                </Box>
+              );
+            }}
+          >
+            {options.map((o, i) => (
+              <StyledMenuItem key={o.id} tabIndex={i} value={o.label}>
+                <ListItemText primary={o.label} />
+                <ListItemIcon
+                  sx={{
+                    display: value.indexOf(o.label) > -1 ? 'block' : 'none',
+                  }}
+                >
+                  <CheckIcon fontSize="small" />
```

Comment:
This icon is slightly too big and causing the select to grow taller and taller

Reply:
its was actually and issue with the `ListItemIcon` component, so I fixed the height of the component above on line 92